### PR TITLE
EN-2381-PAM-process-task-detail-widget

### PR DESCRIPTION
**Pull request by alepintus**
Hi @joewhite101 ,
this is the cherry-pick from [this PR](https://github.com/entando/entando-components/pull/461),
Could you please have a look?
Thanks!

### CODE REVIEW

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by alepintus** [comment]
retest this please

**Comment by entando-service** [comment]
All is Well

File: plugins/entando-plugin-jpkiebpm/src/main/webapp/resources/plugins/jpkiebpm/static/js/custom-data-table.js [hunk 1]
```diff
@@ -127,4 +139,79 @@ org.entando.datatable.CustomDatatable = function (items, idTable, extraConfig, c
...
+                        detailRows.push(tr.attr('id'));
+                    }
+                }
+            }
+    );
+
+
+
+    
+//    // Handle click on "Expand All" button
```
Comment (joewhite101) [inline plugins/entando-plugin-jpkiebpm/src/main/webapp/resources/plugins/jpkiebpm/static/js/custom-data-table.js]: If this is all dead code lets remove it.

File: plugins/entando-plugin-jpkiebpm/src/main/webapp/WEB-INF/plugins/jpkiebpm/aps/jsp/widgets/bpm-datatable-task-list.jsp [hunk 2]
```diff
@@ -168,66 +192,100 @@
...
-        ];
-        var extraConfig = {
+            ];
+            var extraConfig = {
             buttons: extraBtns,
-            onClickRow: function (ev, rowData) {
-                $('#bpm-task-list-modal-data-table-tbody').empty();
-                var url = context + "taskDetail.json?configId=" + configId +"&containerId=" + rowData.containerId + "&taskId=" + rowData.id;
+                    onClickRow: function (event, rowData) {
+                     //   $("#data-table-task-list tbody").on('click', 'tr td:not(:first-child)',function () {
```
Comment (joewhite101) [inline plugins/entando-plugin-jpkiebpm/src/main/webapp/WEB-INF/plugins/jpkiebpm/aps/jsp/widgets/bpm-datatable-task-list.jsp]: Remove if dead

File: plugins/entando-plugin-jpkiebpm/src/main/webapp/resources/plugins/jpkiebpm/static/js/custom-data-table.js [hunk 3]
```diff
@@ -29,6 +35,12 @@ org.entando.datatable.CustomDatatable = function (items, idTable, extraConfig, c
                 targets: i + 1
             };
         });
+//        columns.unshift({
```
Comment (joewhite101) [inline plugins/entando-plugin-jpkiebpm/src/main/webapp/resources/plugins/jpkiebpm/static/js/custom-data-table.js]: Remove if dead

**Comment by alepintus** [comment]
Hi @joewhite101 ,
thanks for the review!
Commented code removed.

Thanks!

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by alepintus** [comment]
Retest this please

**Comment by entando-service** [comment]
All is Well

**Pull request merged**